### DISCUSSION TITLE
change the order of orders

### DISCUSSION THREAD

**Discussion by aristodemos2008** [ho-dev/HattrickOrganizer#1036] [Ideas]
In the "lineup" -> "match orders" screen there should be buttons to change the order of the orders. Sometimes I have several orders in the same minute and the order by which they are processed is relevant.

**Comment by akasolace** [comment]
this is the good idea, could you create a new "feature" ticket 
thank you

**Comment by akasolace** [comment]
has been implemented via [feature #1039](https://github.com/akasolace/HO/issues/1039)
